Makefile

```diff
@@ -46,8 +46,7 @@ E2E_FRAMEWORK_DIR := test/framework
 CAPD_DIR := test/infrastructure/docker
 RELEASE_NOTES_BIN := bin/release-notes
 RELEASE_NOTES := $(TOOLS_DIR)/$(RELEASE_NOTES_BIN)
-LINK_CHECKER_BIN := bin/liche
-LINK_CHECKER := $(TOOLS_DIR)/$(LINK_CHECKER_BIN)
+LINK_CHECKER := docker run -t --rm -v "$$(pwd)\:/capi" becheran/mlc:0.13.4 mlc
```

Comment:
I'd rather not run a third-party image if we can avoid it. WDYT @vincepri @detiber?

Reply:
What's the difference between a third party image and third-party go code?

Reply:
This image doesn't have any details about what's inside it / how it was built (https://hub.docker.com/r/becheran/mlc). I assume it comes from https://github.com/becheran/mlc but there are no guarantees. Maybe I'm being overly cautious?

Reply:
+1, I'd rather build the binary from a specific source then using a custom docker image

Reply:
The linked repository seems pretty new, I'm not 100% comfortable to download a binary from a not-known tool to everyone's computer.

/hold

Reply:
Lemme see what I can do